core/templates/pages/admin-page/admin-page.component.spec.ts

```diff
@@ -0,0 +1,142 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview UnitTests for Admin Page component.
+ */
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ChangeDetectorRef, NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { PlatformFeatureService } from 'services/platform-feature.service';
+import { AdminPageComponent } from './admin-page.component';
+import { AdminRouterService } from './services/admin-router.service';
+
+class MockWindowRef {
+  nativeWindow = {
+    confirm() {
+      return true;
+    },
+    location: {
+      hostname: 'hostname',
+      href: 'href',
+      pathname: 'pathname',
+      search: 'search',
+      hash: 'hash'
+    },
+    open() {
+      return;
+    },
+    onhashchange() {
+      return;
+    }
+  };
+}
+
+describe('Admin misc tab component ', () => {
```

Comment:
Please check the name of the component for both files.

Reply:
done